test/config_parser_SUITE.erl

```diff
@@ -1810,7 +1810,11 @@ mod_jingle_sip(_Config) ->
     ?errf(T(Base#{<<"sdp_origin">> => <<"aaaaaaaaa">>})).
 
 mod_keystore(_Config) ->
```

Comment:
Minor: I think it would be best to have one test case for `mod_keystore` and a separate one for `mod_keystore_keys`. The former would just test the top-level `key_size` option while the latter would check both types of `keys`. I think it would be more readable that way. This can be done separately or in this PR, your decision.